apps/datahub/src/app/app.component.html

```diff
@@ -1,6 +1,11 @@
 <div gnUiSearchStateContainer="mainSearch">
   <header>
-    <datahub-header></datahub-header>
+    <datahub-header
```

Comment:
Hum, do you mean to move the `header-record` from the datahub app to the feature record lib? And the `header` would stay in the datahub app? If I see right, this might make the use of the same header image a bit more cumbersome.

Reply:
I think that the use case has changed, search page and record page should be 2 separated components, actually, both binded to a route.
Initially, we were sharing the header, but actually we are not anymore.
So instead of having a if to switch the header, then another if to switch the content (search or record), we should just have one if at the top, and the components should be refactored in that regard.